src/Interactor/LoginInteractor.php

```diff
@@ -118,7 +118,12 @@ protected function handlePasswordIncorrect(User $user)
             );
         }
 
-        return LoginResponse::passwordIncorrect($user);
+        return $user->isActive() ? LoginResponse::passwordIncorrect($user) : LoginResponse::notActive($user);
```

Comment:
I guess for completeness should mention this response change in the changelog too.

Reply:
Done